es/resource_elasticsearch_opendistro_user.go

```diff
@@ -25,18 +25,16 @@ func resourceElasticsearchOpenDistroUser() *schema.Resource {
 				Required: true,
 			},
 			"password": {
-				Type:             schema.TypeString,
-				Optional:         true,
-				Sensitive:        true,
-				DiffSuppressFunc: onlyDiffOnCreate,
-				ConflictsWith:    []string{"password_hash"},
```

Comment:
Why remove this? I think it still applies?

Reply:
Hi @john-delivuk-rl, just following up on this question, any thoughts?

Reply:
Sorry was on leave last week. I'll take a look shortly and update you.

Reply:
Hi @john-delivuk-rl, following up on this.